src/js/stores/DCOSStore.js

```diff
@@ -26,50 +27,42 @@ class DCOSStore extends EventEmitter {
 
     this.data = {
       marathon: new ServiceTree(),
-      mesos: {
-        frameworks: []
-      },
+      mesos: new SummaryList(),
       dataProcessed: false
     };
 
     this.proxyListeners = [
```

Comment:
hmm we didn't add that level of magic, but we did create methods where you can actually do that yourself. @MatApple probably knows best here since he created it